### PR TITLE
[IMP] l10n_ar: taxes

### PR DESCRIPTION
**Pull request by malb-odoo**
We want to invigorate a Tax Name Taxonomy so that tax names are as codified as possible. This allow for a better display on Invoice Description, and allowed us to implement a smart name_search.

In this PR, we change the taxes name so that it's more clear for the users

task-id: 3052677

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ar/data/template/account.tax-ar_base.csv [hunk 1]
```diff
@@ -1,97 +1,97 @@
...
+ri_tax_percepcion_iibb_rn_sufrida,"1% IIBB R N I","Perception IIBB Río Negro incurred","Perc IIBB Río Negro S","4","fixed","1.0","purchase","tax_group_percepcion_iibb_rn","base","invoice","","Percepción IIBB Río Negro Sufrida"
+"","","","","","","","","","tax","invoice","base_percepcion_iibb_rn_sufrida",""
+"","","","","","","","","","base","refund","",""
+"","","","","","","","","","tax","refund","base_percepcion_iibb_rn_sufrida",""
+ri_tax_percepcion_iibb_az_sufrida,"1% IIBB S C I","Perception IIBB Santa Cruz incurred","Perc IIBB Santa Cruz S","4","fixed","1.0","purchase","tax_group_percepcion_iibb_az","base","invoice","","Percepción IIBB Santa Cruz sufrida"
+"","","","","","","","","","tax","invoice","base_percepcion_iibb_az_sufrida",""
+"","","","","","","","","","base","refund","",""
+"","","","","","","","","","tax","refund","base_percepcion_iibb_az_sufrida",""
+ri_tax_percepcion_iibb_tf_sufrida,"0% IIBB T F I","Perception IIBB Tierra del Fuego incurred","Perc IIBB Tierra del Fuego S","4","fixed","1.0","purchase","tax_group_percepcion_iibb_tf","base","invoice","","Percepción IIBB Tierra del Fuego sufrida"
```
Comment (oco-odoo) [inline addons/l10n_ar/data/template/account.tax-ar_base.csv:94]: Same here, it's not a 0% tax ; it's fixed with an amount of 1.0

**Review by oco-odoo — CHANGES_REQUESTED** [review]

File: addons/l10n_ar/data/template/account.tax-ar_ex.csv [hunk 2]
```diff
@@ -1,109 +1,109 @@
...
+ri_tax_percepcion_iva_aplicada,"0%","Perception VAT applied","Perc VAT A","4","False","fixed","0.0","sale","tax_group_percepcion_iva","base","invoice","","Percepción IVA Aplicada"
+"","","","","","","","","","","tax","invoice","ri_percepcion_iva_aplicada",""
+"","","","","","","","","","","base","refund","",""
+"","","","","","","","","","","tax","refund","ri_percepcion_iva_aplicada",""
+ri_tax_percepcion_ganancias_aplicada,"0% A E","Perception of Applied Earnings","Perc Profit A","4","False","fixed","0.0","sale","tax_group_percepcion_ganancias","base","invoice","","Percepción Ganancias Aplicada"
+"","","","","","","","","","","tax","invoice","ri_percepcion_ganancias_aplicada",""
+"","","","","","","","","","","base","refund","",""
+"","","","","","","","","","","tax","refund","ri_percepcion_ganancias_aplicada",""
+ri_tax_percepcion_ganancias_sufrida,"0% E I","Perception Earnings incurred","Perc Profit S","4","","fixed","1.0","purchase","tax_group_percepcion_ganancias","base","invoice","","Percepción Ganancias Sufrida"
```
[the remark this answers is not in the document]
Comment (malb-odoo) [reply addons/l10n_ar/data/template/account.tax-ar_ex.csv:10]: Yep you're right it's a mistake, my bad will correct that now :+1:

File: addons/l10n_ar/data/template/account.tax-ar_base.csv [hunk 3]
```diff
@@ -1,97 +1,97 @@
...
+"","","","","","","","","","tax","refund","base_percepcion_iibb_se_sufrida",""
+ri_tax_percepcion_iibb_tn_sufrida,"1% IIBB T I","Perception IIBB Tucumán incurred","Perc IIBB Tucumán S","4","fixed","1.0","purchase","tax_group_percepcion_iibb_tn","base","invoice","","Percepción IIBB Tucumán sufrida"
+"","","","","","","","","","tax","invoice","base_percepcion_iibb_tn_sufrida",""
+"","","","","","","","","","base","refund","",""
+"","","","","","","","","","tax","refund","base_percepcion_iibb_tn_sufrida",""
+ri_tax_percepcion_iibb_ha_sufrida,"1% IIBB Chaco I","Perception IIBB Perception Chaco incurred","Perc IIBB Chaco S","4","fixed","1.0","purchase","tax_group_percepcion_iibb_ha","base","invoice","","Percepción IIBB Chaco Sufrida"
+"","","","","","","","","","tax","invoice","base_percepcion_iibb_ha_sufrida",""
+"","","","","","","","","","base","refund","",""
+"","","","","","","","","","tax","refund","base_percepcion_iibb_ha_sufrida",""
+ri_tax_percepcion_iibb_ct_sufrida,"1% IIBB C I","Perception IIBB Chubut incurred","Perc IIBB Chubut S","4","fixed","1.0","purchase","tax_group_percepcion_iibb_ct","base","invoice","","Percepción IIBB Chubut sufrida"
```
Comment (oco-odoo) [inline addons/l10n_ar/data/template/account.tax-ar_base.csv:66]: Name clash with ri_tax_percepcion_iibb_co_sufrida :/

File: addons/l10n_ar/data/template/account.tax-ar_base.csv [hunk 4]
```diff
@@ -1,97 +1,97 @@
...
+"","","","","","","","","","tax","invoice","base_percepcion_iibb_ct_sufrida",""
+"","","","","","","","","","base","refund","",""
+"","","","","","","","","","tax","refund","base_percepcion_iibb_ct_sufrida",""
+ri_tax_percepcion_iibb_fo_sufrida,"1% IIBB F I","Perception IIBB Formosa Formosa incurred","Perc IIBB Formosa S","4","fixed","1.0","purchase","tax_group_percepcion_iibb_fo","base","invoice","","Percepción IIBB Formosa Sufrida"
+"","","","","","","","","","tax","invoice","base_percepcion_iibb_fo_sufrida",""
+"","","","","","","","","","base","refund","",""
+"","","","","","","","","","tax","refund","base_percepcion_iibb_fo_sufrida",""
+ri_tax_percepcion_iibb_mi_sufrida,"1% IIBB M I","Perception IIBB Misiones incurred","Perc IIBB Misiones S","4","fixed","1.0","purchase","tax_group_percepcion_iibb_mi","base","invoice","","Percepción IIBB Misiones Sufrida"
```
Comment (oco-odoo) [inline addons/l10n_ar/data/template/account.tax-ar_base.csv:74]: Name clash with ri_tax_percepcion_iibb_za_sufrida

File: addons/l10n_ar/data/template/account.tax-ar_ex.csv [hunk 5]
```diff
@@ -1,109 +1,109 @@
...
+ri_tax_percepcion_iibb_se_aplicada,"0% IIBB S E","Perception IIBB Santiago del Estero applied","Perc IIBB Santiago del Estero A","4","False","fixed","0.0","sale","tax_group_percepcion_iibb_se","base","invoice","","Percepción IIBB Santiago del Estero Aplicada"
+"","","","","","","","","","","tax","invoice","ri_percepcion_iibb_se_aplicada",""
+"","","","","","","","","","","base","refund","",""
+"","","","","","","","","","","tax","refund","ri_percepcion_iibb_se_aplicada",""
+ri_tax_percepcion_iibb_tn_aplicada,"0% IIBB T","Perception IIBB Tucumán applied","Perc IIBB Tucumán A","4","False","fixed","0.0","sale","tax_group_percepcion_iibb_tn","base","invoice","","Percepción IIBB Tucumán Aplicada"
+"","","","","","","","","","","tax","invoice","ri_percepcion_iibb_tn_aplicada",""
+"","","","","","","","","","","base","refund","",""
+"","","","","","","","","","","tax","refund","ri_percepcion_iibb_tn_aplicada",""
+ri_tax_percepcion_iibb_ha_aplicada,"0% IIBB C","Perception IIBB Chaco applied","Perc IIBB Chaco A","4","False","fixed","0.0","sale","tax_group_percepcion_iibb_ha","base","invoice","","Percepción IIBB Chaco Aplicada"
```
Comment (oco-odoo) [inline addons/l10n_ar/data/template/account.tax-ar_ex.csv:74]: Name clash with ri_tax_percepcion_iibb_co_aplicada

File: addons/l10n_ar/data/template/account.tax-ar_ex.csv [hunk 6]
```diff
@@ -1,109 +1,109 @@
...
+"","","","","","","","","","","tax","refund","ri_percepcion_iibb_ha_aplicada",""
+ri_tax_percepcion_iibb_ct_aplicada,"0% IIBB CB","Perception IIBB Chubut applied","Perc IIBB Chubut A","4","False","fixed","0.0","sale","tax_group_percepcion_iibb_ct","base","invoice","","Percepción IIBB Chubut Aplicada"
+"","","","","","","","","","","tax","invoice","ri_percepcion_iibb_ct_aplicada",""
+"","","","","","","","","","","base","refund","",""
+"","","","","","","","","","","tax","refund","ri_percepcion_iibb_ct_aplicada",""
+ri_tax_percepcion_iibb_fo_aplicada,"0% IIBB F","Perception IIBB Formosa applied","Perc IIBB Formosa A","4","False","fixed","0.0","sale","tax_group_percepcion_iibb_fo","base","invoice","","Percepción IIBB Formosa Aplicada"
+"","","","","","","","","","","tax","invoice","ri_percepcion_iibb_fo_aplicada",""
+"","","","","","","","","","","base","refund","",""
+"","","","","","","","","","","tax","refund","ri_percepcion_iibb_fo_aplicada",""
+ri_tax_percepcion_iibb_mi_aplicada,"0% IIBB M","Perception IIBB Misiones applied","Perc IIBB Misiones A","4","False","fixed","0.0","sale","tax_group_percepcion_iibb_mi","base","invoice","","Percepción IIBB Misiones Aplicada"
```
Comment (oco-odoo) [inline addons/l10n_ar/data/template/account.tax-ar_ex.csv:86]: Name clash with ri_tax_percepcion_iibb_za_aplicada

**Review by oco-odoo — CHANGES_REQUESTED** [review]
few more things ... Blame bib ! :p

**Comment by oco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**